### ISSUE TITLE
Error while deploying

### ISSUE THREAD

**Issue by meresey** [oracle-quickstart/oci-cis-landingzone-quickstart#61]
Encountering error below when i execute apply

![Screenshot 2022-05-25 124116](https://user-images.githubusercontent.com/20772780/170232809-119e4b93-4d9e-40f0-8ca2-9ee0984aa7c7.jpg)

**Comment by andrecorreaneto** [comment]
Hello @meresey, sorry for not getting back earlier, but is this still an issue? In which environment were you running terraform?

**Comment by gillfimj** [comment]
Yes this is still an issue. I just ran  into it today 10/10/2022. I am running terraform in Windows:

```text
terraform version
Terraform v1.2.7
on windows_amd64
+ provider registry.terraform.io/hashicorp/null v3.1.1
+ provider registry.terraform.io/oracle/oci v4.96.0

Your version of Terraform is out of date! The latest version
is 1.3.2. You can update by downloading from https://www.terraform.io/downloads.html

oci --version
3.15.2
```

**Comment by gillfimj** [comment]
I have been able to get past this error by modifying the source as follows:

```terraform
#     command = "sleep ${local.delay_in_secs}" # Wait for keys policy to be available.
    command = "start-sleep ${local.delay_in_secs}" # Wait for compartments to be available.
    interpreter = ["PowerShell", "-Command"]
```

It would be nice if there was a better way to handle this to make it more dynamic based on the environment of the developer, i.e. if Linux do this, if Windows do that. Not sure if that's possible.

**Comment by andrecorreaneto** [comment]
One option is using git bash for a linux-like experience in Windows. With that said, we'll look into how to provide a seamless native Windows experience.

**Comment by gillfimj** [comment]
I should further explain the Windows environment in which this happens. I do all my coding in VSCode, so the errors are happening in the PowerShell terminal. I get the idea of using git bash, but changing focus just to run terraform is quite a hassle.

**Comment by Halimer** [comment]
This should be resolved in release 2.5.1 we just released
